puppet/zulip/manifests/postgres_appdb_base.pp

```diff
@@ -48,7 +48,8 @@
 
   # We bundle a bunch of other sysctl parameters into 40-postgresql.conf
   file { '/etc/sysctl.d/30-postgresql-shm.conf':
-    ensure => absent,
+    ensure  => absent,
+    require => Package[$postgresql],
   }
```

Comment:
I wonder if we should just not be doing this at all.  One my Bionic system, that file only contains comments:

```
tabbott@coset:~$ cat /etc/sysctl.d/30-postgresql-shm.conf 
# Shared memory settings for PostgreSQL on Linux

# The settings here are upper limits; performance is not affected if the
# settings are larger than necessary. If other programs use shared memory as
# well, you will have to coordinate the size settings between them.

# Maximum size of a single shared memory segment in bytes
#kernel.shmmax = 33554432
# Maximum total size of all shared memory segments in pages (normally 4096 bytes)
#kernel.shmall = 2097152

```

Reply:
Yeah, I bet we can just drop it.  Updatein incoming.